parser-typechecker/src/Unison/PrettyPrintEnv.hs

```diff
@@ -35,12 +34,14 @@ instance Show PrettyPrintEnv where
 
 fromNames :: Int -> Names -> PrettyPrintEnv
 fromNames len names = PrettyPrintEnv terms' types' where
-  terms' r = shortestName . Set.map HQ'.toHQ $ Names.termName len r names
-  types' r = shortestName . Set.map HQ'.toHQ $ Names.typeName len r names
+  terms' r = shortestName . Set.map Name.convert $ Names.termName len r names
+  types' r = shortestName . Set.map Name.convert $ Names.typeName len r names
```

Comment:
Gratuitous change, but trying to move away from the special purpose conversion functions I can never remember. Just use `Name.convert` or `Name.parse`, and add a type annotation at the call site if necessary for clarity.

Reply:
I don’t think I’ve learned this yet and may need reminders in the future!